.github/workflows/tests.yml

```diff
@@ -22,8 +23,6 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
       run: |
-        sudo apt update
-        sudo apt install cmake clang libprotobuf-dev protobuf-compiler
```

Comment:
Linux requires installing some packages

Reply:
That I will add.... just removed it...so it should work on windows